README.md

```diff
@@ -38,6 +52,63 @@ for chunk in stream:
   print(chunk['message']['content'], end='', flush=True)
 ```
 
+## Custom client
+
+A custom client can be created with the following fields:
+
+- `host`: The Ollama host (default: `http://localhost:11434`)
+- `timeout`: The timeout for requests (default: `None`)
+- `follow_redirects`: Whether to follow redirects (default: `True`)
+- `headers`: Additional headers to send with requests (default: `{}`)
+
+```python
+from ollama import Client
+# default client
+client = Client()
+# or custom client
```

Comment:
This section is on custom clients so mentioning the default client seems unnecessary